### PR TITLE
Add January updates

### PR DESCRIPTION
**Pull request by astrojuanlu**
Featuring @s-m-e and @JackCrusoe47 work too! 🎉

### CODE REVIEW

File: content/articles/2022-02-january-updates.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+Title: December updates
```
Comment (Yash-10) [inline content/articles/2022-02-january-updates.md]:
```suggestion
Title: January updates
```
:sweat_smile:

File: content/articles/2022-02-january-updates.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Date: 2021-02-01 16:00 CET
```
Comment (Yash-10) [inline content/articles/2022-02-january-updates.md]:
```suggestion
Date: 2022-02-01 16:00 CET
```

File: content/articles/2022-02-january-updates.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+Tags: NumFOCUS,Updates
+slug: january-updates
+lang: en
+Author: Juan Luis Cano Rodríguez
+
+This January we have spent a lot of time discussing Sebastian proposal
```
Comment (Yash-10) [inline content/articles/2022-02-january-updates.md]:
```suggestion
This January, we spent a lot of time discussing Sebastian's proposal
```

File: content/articles/2022-02-january-updates.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+to [add a new `OrbitArray` object](https://github.com/poliastro/poliastro/pull/1445)
+in line with our Small Development Grant.
+We are now all in the same page and the implementation is progressing nicely!
```
Comment (Yash-10) [inline content/articles/2022-02-january-updates.md]:
```suggestion
We are now all on the same page and the implementation is progressing nicely!
```

**Review by Yash-10 — CHANGES_REQUESTED** [review]
LGTM! Only a few comments.

**Comment by astrojuanlu** [comment]
Wow, too many typos 😅 Thanks @Yash-10 !

**Pull request merged**